Gemfile.lock

```diff
@@ -153,4 +153,4 @@ DEPENDENCIES
   rails (>= 5.0)
 
 BUNDLED WITH
-   2.0.2
+   2.1.4
```

Comment:
Is this normal? Would it cause issues?

Reply:
Ah, that's just an artifact of my local env. That said, will remove during rebase.